quality_assurance_template.md

```diff
@@ -0,0 +1,107 @@
+* [ ] Educational content ends with a section of additional resources
+* [ ] Final section is Feedback.
+  * [ ] Learning objectives linked from YAML rather than rewritten
+  * [ ] Feedback link is updated appropriately to automatically fill in the module name when clicked by learner.
+
+## Content
+
+* [ ] Complexity of the material covered seems appropriate given prerequisites.
+* [ ] Time estimate appears accurate for a learner of the targeted level.
+* [ ] Clearly defined learning objectives using strong, descriptive verbs. (See [Bloom's taxonomy](https://cft.vanderbilt.edu/guides-sub-pages/blooms-taxonomy/) for ideas.)
+* [ ] Every learning objective is covered in the module content.
+* [ ] All major topics covered are represented by a learning objective -- there's no significant knowledge imparted that isn't specified in the learning objectives
+* [ ] The module title, blurb/comment, and long description accurately reflect the content of the module.
+* [ ] Headers are informative and follow a sensible hierarchical structure (the TOC in the left margin should give a good overview of the content covered)
+* [ ] Avoids unclear language: unexplained idioms or references, unexplained acronyms, unnecessary technical language.
+* [ ] Unusual words, or words taking on a very specific meaning in context, are always defined for the learner, either on the page (e.g. using footnotes) or with links to a definition/glossary.
+* [ ] Provides pronunciation guides for any especially unusual words of particular importance (a common example is package names, such as dplyr)
+* [ ] Avoids unnecessarily gendered language (e.g. uses "they" singular rather than "he or she" for an unknown person).
+* [ ] Informative link text (e.g. instead of "To learn more about python, click [here](www.example.com)", say "Read this article to [learn more about python](www.example.com).")
+* [ ] Uses specially formatted highlight boxes consistently and appropriately
+* [ ] Short, digestible pieces --- avoids long paragraphs and breaks long sections up with sub-headers
+* [ ] All links work (See [Branch References to Change prior to PR Approval](#Branch-References-to-Change-prior-to-PR-Approval) for a place to keep track of internal references that will need to be updated.)
+* [ ] Spelling and grammar are correct.
+
+
+## Code availability
+If the module includes code that learners may want to run:
+* [ ] A link is provided to run the code in binder
+* [ ] A link to the raw code and any other necessary materials (e.g., data) is provided for learners who prefer to work on their own machine
+
+## Formative assessment
+
+* [ ] Frequent [formative assessment](https://carpentries.github.io/instructor-training/02-practice-learning/#identifying-and-correcting-misconceptions) in the form of knowledge checks (referred to as quizzes) and/or hands-on exercises
```

Comment:
to avoid confusion, maybe just "in the form of quizzes and/or hands-on exercises"?

Reply:
yeah good point! I think I was trying too hard to not totally change previous language, but that doesn't seem to be a sensible thing to do in this instance!